pgjdbc/src/main/java/org/postgresql/core/ByteOptimizedUTF8Encoding.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2003, PostgreSQL Global Development Group
```

Comment:
The year should be current. This file did not exist in 2003

Reply:
done